bindings/rust/lib.rs

```diff
@@ -35,10 +35,10 @@ pub const NODE_TYPES: &'static str = include_str!("../../src/node-types.json");
 
 // Uncomment these to include any queries that this grammar contains
 
-pub const HIGHLIGHTS_QUERY: &'static str = include_str!("../../queries/highlights.scm");
+// pub const HIGHLIGHTS_QUERY: &'static str = include_str!("../../queries/highlights.scm");
```

Comment:
```suggestion
pub const HIGHLIGHTS_QUERY: &'static str = include_str!("../../queries/highlights.scm");
```
I assume these were commented out by the regeneration machinery? At any rate, I think we want to include these, as they do actually exist.